contracts/makefile

```diff
@@ -136,7 +136,7 @@ deploy-base-anvil :;
 
 # THIS IS FOR TESTING ONLY
 deploy-base-anvil-ledger :;
-	@$(MAKE) deploy-any-local-ledger context=local_multi rpc=base_anvil hd_path="m/44\'/60\'/1\'/0/0" sender=${PRODUCTION_ADDRESS}
+	@$(MAKE) deploy-any-local-ledger context=local_multi rpc=base_anvil hd_path="m/44\'/60\'/1\'/0/0" sender=${SENDER_ADDRESS}
```

Comment:
This hd_path historically has not worked for me. My wallet uses a different path than the above. I've had to remove the path and supply `--ledger` argument, which prompts for a sig on my hardware device.

Reply:
right, I'm siding towards just allowing the path to be defined through an env variable